### PR TITLE
perf: enable built-in AI to set usage status and dict option

### PR DESCRIPTION
**Pull request by tisfeng**
image

### CODE REVIEW

File: Easydict/Swift/Feature/Configuration/DefaultKeys.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//
+//  DefaultKeys.swift
+//  Easydict
+//
+//  Created by tisfeng on 2024/4/28.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Foundation
```
Comment (phlpsong) [inline Easydict/Swift/Feature/Configuration/DefaultKeys.swift:9]: What is this file for?

Comment (tisfeng) [reply Easydict/Swift/Feature/Configuration/DefaultKeys.swift]: Just forgot to delete it.

**Comment by tisfeng** [comment]
I just refactored some code related to OpenAI, mainly allowing dynamic keys to be created based on the type of service, and then storing the configuration items for that service. This has reduced a lot of duplicated code.

@phlpsong  There is still room for further optimization in the code here, like `Defaults.Keys` in the `Configuration+Defaults` file, these can be handled in later issues, such as #479 and #497.

**Comment by tisfeng** [comment]
I'm going to merge this PR, are there any questions?

**Review by phlpsong — APPROVED** [review]

**Comment by phlpsong** [comment]
Nice work👍

I have been busy recently, I will schedule some time to take a look at these optimization issues later.

**Comment by tisfeng** [comment]
ok

**Pull request merged**